### ISSUE TITLE
Reconnection after disconnect due to bad network connectivity ended up with one-way video call

### ISSUE THREAD

**Issue by michaelkaye** [vector-im/element-call#1890]
### Steps to reproduce

1. Using call.element.io with a legacy room (no e2e secret provided in the URL) with two people in the call.
2. Have a call on a network with some problems: voice and video were occasionally interrupted.
3. I saw the remote video freeze, associated with a total network issue on my machine (element-web cut off, etc) 
4. I returned by refreshing the browser
5. I could see the voice and video of the other party, but they could only see a black screen from me.
6. Refreshing on my side again did not help
7. Moving to a new call URL (now with e2e secret) fixed it.

#### What did you expect?
Reconnection should get back to two way sound & video

#### What happened instead?
Reconnection caused one-way sound & video

### URL for webapp

call.element.io

### Will you send logs?

Yes

**Comment by dbkr** [comment]
I assume you could see yourself in the video preview? There are more 'awaiting getUserMedia' lines than I would expect but otherwise I'm struggling to see much from the logs.

**Comment by michaelkaye** [comment]
Yes, i could see both videos fine (and no clones or duplicates on my side)
